sample-operators/mysql-schema/src/main/java/io/javaoperatorsdk/operator/sample/MySQLSchemaReconciler.java

```diff
@@ -54,6 +54,13 @@ public void initContext(MySQLSchema primary, Context context) {
     context.put(MYSQL_SECRET_USERNAME, userName);
   }
 
+  private String randomPassword(int length) {
+    SecureRandom random = new SecureRandom(); // Compliant for security-sensitive use cases
```

Comment:
We shouldn't need to do this, though… We need a way to exclude such false positive from Sonar.

Reply:
Did not find a way, not sure if possible for the security issues.